src/bci_build/package/base/README.md.j2

```diff
@@ -0,0 +1,30 @@
+# SLE BCI {{ image.os_version.pretty_os_version_no_dash }} Base Container Image
+{% include 'badges.j2' %}
+
+## Description
+
+SUSE Linux Enterprise Base Container Images (SLE BCI) provide truly open,
+flexible, and secure container images and application development tools. The
+images consist of container environments based on SUSE Linux Enterprise and
+designed to be a secure base for any containerized workload.
+
+SLE BCI is freely available, re-distributable, and supported across many
+different environments. These templates and tools address modern, containerized
+application development and CI/CD application containerization.  They can be
```

Comment:
what can be used? BCI? the templates and tools from the previous sentence? which is also not clear.